src/types/dataModel/backgroundCheckWebhookPayload.ts

```diff
@@ -0,0 +1,14 @@
+import { z } from 'zod';
+
+const zBackgroundCheckWebhookPayload = z.object({
+  data: z.object({
+    employee_email: z.string(),
+    overall_status: z.string(),
```

Comment:
also, can you make this field `z.union([z.literal("PASSED"), z.literal("FAILED")])` I am guessing the request from the webhook will have them as all uppercase since that was the example.